### PR TITLE
Changed instances of furlough in code to absent

### PR DESCRIPTION
**Pull request by Christian-QA**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -26,21 +26,21 @@ $hmrc-assets-path: "/coronavirus-job-support-scheme-calculator/assets/lib/hmrc-f
   line-height: 1.09;
 }
 
-.furlough-heading {
+.absent-heading {
```
Comment (mikail-khan) [inline app/assets/stylesheets/application.scss:29]: why did you choose this name?

Comment (Christian-QA) [reply app/assets/stylesheets/application.scss:29]: Furlough is essentially a leave of absence, so went with absent to keep it to one word.

Comment (sureshhmrc) [reply app/assets/stylesheets/application.scss:29]: hmm, absent word looks weird though... may be you can use `grant` or something

Comment (Christian-QA) [reply app/assets/stylesheets/application.scss:29]: Alright, I'll go with grant

**Pull request closed without merging**